lms/djangoapps/learner_dashboard/urls.py

```diff
@@ -0,0 +1,11 @@
+"""
+Learner's Dashboard urls
+"""
+
+from django.conf.urls import patterns, url
+from . import views
+
+urlpatterns = patterns(
+    '',
+    url(r'^programs', views.view_programs, name="program_listing_view")
```

Comment:
Nitpick: as recommended by PEP 8, pick a quote level and stick with it. Most of the application uses single quotes.